examples/python/gRPC/util/fb_to_dict.py

```diff
@@ -137,6 +138,7 @@ def fb_flatc_dict(fb_obj: bytearray, schema_file_name: SchemaFileNames) -> Dict:
             tmp_json = Path(f.name + ".json")
 
         sp_ret_code = flatc_proc.wait()
+        os.sync()
```

Comment:
What about something like this? Forcing the write of the FlatBuffer by closing the file?

```python
try:
  with tempfile.NamedTemporaryFile() as f:
      # store the flatbuffer in a temporary file
      with f.file as tmp_file:
          tmp_file.write(fb_obj) 
  
      flatc_proc = sp.Popen(
          ["flatc", "--json", "--raw-binary", "--strict-json", schema_path, "--", f.name], cwd="/tmp"
      )
      
      fb_json_path = Path(f.name + ".json")
      sp_ret_code = flatc_proc.wait()
  
      if sp_ret_code != 0 or not fb_json_path.is_file():
          raise ChildProcessError(
              f"A problem occured during flatbuffers object to json conversion using flatc!\
              The file {fb_json_path} was not properly created. Has the fbs schema file the `root_type` directive set?"
          )
      
      # load json representation of the flatbuffers object
      with open(fb_json_path, "r") as json_file:
          json_dict = json.loads(json_file.read())
finally:
  fb_json_path.unlink()
```

Reply:
The problem here is not that the temporary file is not written, but the json file that flatc creates.
A Child error is raised, due to  `fb_json_path.is_file()` returning `False` in the if statement here:
https://github.com/agri-gaia/seerep/blob/b3824b1a7b0e232218e39d4ed816086e77bd992f/examples/python/gRPC/util/fb_to_dict.py#L143